js/ad_manager_template.js

```diff
@@ -217,6 +220,10 @@ OO.Ads.manager(function(_, $) {
 
     };
 
+    this.onVideoTagFound = function() {
```

Comment:
Should add jsdoc here. Also, would it be better to add the arguments to the function signature?